### PR TITLE
minor fix - point to issues not PRs

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

File: source/releases/development/1.3.14.rst [hunk 1]
```diff
@@ -12,10 +12,10 @@ Improvements
 Bugfixes
 --------
 
-* Fixed crash when removing a Spline vertex using "Remove item (Smart)" (fix `#1188 <https://github.com/synfig/synfig/pull/1188>`_).
+* Fixed crash when removing a Spline vertex using "Remove item (Smart)" (`issue #1102 <https://github.com/synfig/synfig/issues/1102>`_).
 * Fixed incorrect placement of width points on outline when loading old files (see details `here <https://github.com/synfig/synfig/issues/314#issuecomment-601336943>`_ and `here <https://github.com/synfig/synfig/issues/1284>`_).
 * Fixed crash when Shade Layer has no sublayers (`issue #1272 <https://github.com/synfig/synfig/pull/1272>`_).
-* Fixed popup menu disappearing right after button release for Widget Gradient/Spline (`issue #1277 <https://github.com/synfig/synfig/pull/1277>`_).
+* Fixed popup menu disappearing right after button release for Widget Gradient/Spline (`issue #1272 <https://github.com/synfig/synfig/issues/1272>`_).
```
Comment (morevnaproject) [inline source/releases/development/1.3.14.rst:18]: I guess this should point to 1274? ^__^
https://github.com/synfig/synfig/issues/1274

Comment (rodolforg) [reply source/releases/development/1.3.14.rst:18]: lol

Comment (rodolforg) [reply source/releases/development/1.3.14.rst]: Done

**Comment by morevnaproject** [comment]
Thank you!

**Pull request merged**